assets/js/addAnother.js

```diff
@@ -1,4 +1,14 @@
 document.addEventListener('DOMContentLoaded', () => {
+  const isOffenderMainOccupierCheckbox = document.getElementById('isOffender')
```

Comment:
ah.. so we wouldn't want this page specific code in a generic component.. think it needs moving to a new file.
We can then use "the pattern" to check whether we're on this page rather than fish to see if the specific component exists